migrations/20230704145116-create_users_table.sql

```diff
@@ -0,0 +1,10 @@
+
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS users (
+    user_id SERIAL PRIMARY KEY,
+    github_username TEXT,
+    github_ID INTEGER,
+    UNIQUE (user_id)
```

Comment:
Refer to https://gorm.io/docs/models.html#gorm-Model. We actually need a couple more fields to support the ORM capabilities of GORM.